### DISCUSSION TITLE
A question about LLDP packet forwarding

### DISCUSSION THREAD

**Discussion by mercy34123** [lldpd/lldpd#555] [Q&A]
I want to apply lldpd project to device such as a router or wireless  access point, and i have some questions. A Lldp packet received by the Ethernet interface eth0, i want to know if it will be forwarded by the bridge interface br0 to a wireless interface.If it will be forwarded,i want to konw if there any way to avoid this problem.Otherwise why it will not  be forwarded.

**Comment by vincentbernat** [comment]
LLDP packets are not meant to be forwarded. On Linux, they are not.

**Reply by mercy34123** [reply]
Thank you very much for your answer. I want to know the reason that  LLDP packets are not meant to be forwarded, is there something in the code that restricts LLDP packets to be forwarded？

**Reply by vincentbernat** [reply]
They are using link-local MAC addresses and as per IEEE something, they are not meant to be forwarded. On Linux, this can be changed with `/sys/class/net/${bridge}/bridge/group_fwd_mask`.
